lib/index.js

```diff
@@ -1 +1,10 @@
+document.getElementById('signupLink').addEventListener('click', displaySignupForm)
 
+function displaySignupForm() {
+  document.getElementById('loginButton').style.display = 'none';
+  document.getElementById('signupLink').style.display = 'none';
+  document.getElementById('signupButton').style.display = 'inline';
+  document.getElementById('backLink').style.display = 'block';
+  document.getElementById('confirmPasswordField').style.display = 'inline';
+  document.getElementById('confirmPasswordLabel').style.display = 'block';
```

Comment:
Depending on how big a file gets, for readability and reuse you can consider possible benefits of assigning document.getElementById's first to a variable, such as:
`var signupLink = document.getElementByID('signupLink');`
`signupLink.addEventListener('click', displaySignupForm)`

As is, no worries, this file is definitely readable since it is 9 lines :)

Reply:
Great idea, I will definitely keep this in mind as my file gets bigger and I reuse these elements.